libcloudforensics/providers/kubernetes/workloads.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
```

Comment:
Here and in the new files, might wanna update the date for the copyright while at it